programs/protocol-contracts-solana/src/lib.rs

```diff
@@ -267,13 +273,20 @@ pub mod gateway {
             return err!(Errors::TSSAuthenticationFailed);
         }
 
+        let pda_ata = get_associated_token_address(&pda.key(), &ctx.accounts.mint_account.key());
```

Comment:
Could we use more explicit variable name? `pda_ata` is a bit complicated to understand

Might want to also add a comment to describe what it represent

Reply:
It relates to a "program derived address", which is the way Solana creates addresses for specific programs deployed onchain.

Reply:
2426c0766dfd2073e8fbb792e47ea5ab00c22649

Reply:
Is pda for "program derived address" a generally used acronym in Solana? I think we should avoid using acronyms in general but if standard term that's fine

But ata could be renamed into `pda_associated_token`, using variable names like `pda_ata` really makes code harder to read

Reply:
yes PDA and ATA are both generally used acronyms.